src/main/java/org/opentripplanner/raptor/api/request/RaptorViaConnection.java

```diff
@@ -0,0 +1,133 @@
+package org.opentripplanner.raptor.api.request;
+
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.opentripplanner.framework.time.DurationUtils;
+import org.opentripplanner.raptor.api.model.RaptorConstants;
+import org.opentripplanner.raptor.api.model.RaptorStopNameResolver;
+import org.opentripplanner.raptor.api.model.RaptorTransfer;
+
+/**
+ * A via-connection is used to connect two stops during the raptor routing. The Raptor
+ * implementation uses these connections to force the path through at least one connection per
+ * via-location. This is not an alternative to transfers. Raptor supports several use-cases through
+ * via-connections:
+ *
+ * <h4>Route via a pass-through-stop</h4>
+ * Raptor will allow a path to go through a pass-through-stop. The stop can be visited on-board
+ * transit, or at the alight- or board-stop. The from-stop and to-stop is the same, and the
+ * minimum-wait-time must be zero.
```

Comment:
Can you explain the last sentence? Does it mean that you can only get off and then on again at the very same stop?

Reply:
No, to specify a passThroughStop `from` === `to`- only one visit to the stop is requiered - can be on-board. I will consider changing the design later.